### DISCUSSION TITLE
DAX Studio {version - 3.0.0] results copy to clipboard not working

### DISCUSSION THREAD

**Discussion by anjali-prasad-intel** [DaxStudio/DaxStudio#1021] [Q&A] [answered]
I am clicking Clipboard to copy the results as a table, but once I go to excel and paste, it is pasting the query not the results. 
How do I copy the results as a pivot table? or just as a table?

![image](https://user-images.githubusercontent.com/112016478/211261767-27177527-ba20-4ae1-985f-24525b954813.png)

**Comment by marcosqlbi** [comment, marked as answer]
The Results button specifies where to send the result of the next query you execute. Did you try to run a query after you set Clipboard?

**Reply by anjali-prasad-intel** [reply]
No, I click Results and then clipboard, I directly go to  an excel file and hit CTRL + V and the results from the query doesn't paste.

**Reply by marcosqlbi** [reply]
So you just have to do this:

1. Choose the Results type
2. Click Run
3. Paste the result of the query wherever you want.

**Reply by dgosbell** [reply]
Marco is correct. The other option is that you can use the Results Table output, then: 
1. Click in the grid
2. type Ctrl+A to select all
3. type Ctrl+C to copy
4. Paste the result of the query wherever you want.

**Reply by anjali-prasad-intel** [reply]
Thank you! This worked!
